config/ui-extensions/apirules/details

```diff
@@ -30,6 +30,11 @@ body:
     severity: warning
     source: '"alert.corsPolicy"'
     visibility: '$not($exists(spec.corsPolicy))'
+  - simple: true
+    widget: Alert
+    severity: warning
+    source: '"alert.gateway.details"'
+    visibility: 'metadata.annotations."gateway.kyma-project.io/original-version"="v2alpha1" and $not($exists($match(spec.gateway,/^[a-z0-9_]+(-[a-z0-9_]+)*\/[a-z0-9_]+(-[a-z0-9_]+)*$/))[0])'
```

Comment:
Is this valid? As far as I remember this does not only apply to v2alpha1 APIRules, because we don't want to change validation in v1beta1, but add a warning in the UI. In the task there is this description:

> Implement busola pop up for APIRule v1beta2 details and form view advising to use

I think `v1beta2` is a typo and should be v1beta1

Reply:
so have the warning for all versions?